### DISCUSSION TITLE
如何减少录像的编码次数

### DISCUSSION THREAD

**Discussion by visitewen** [rts-cn/rts#49] [FreeSWITCH]
FS在录像时会对H264进行重新编码然后转发，对资源占用比较高，有的终端可能对重编码的包解析兼容也不是很好，如果两端都是H264的话，这个重新编码是不是可以省掉？

**Comment by seven1240** [comment]
我们在XSwitch里有不转码录像的解决方案。参见： https://xswitch.cn
